packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -1,23 +1,65 @@
-const packageExists = require('./utils/package-exists');
+const path = require('path');
 const webpack = packageExists('webpack') ? require('webpack') : undefined;
-const logger = require('./utils/logger');
 const webpackMerge = require('webpack-merge');
-const { core, coreFlagMap } = require('./utils/cli-flags');
+const { writeFileSync } = require('fs');
+const { options: coloretteOptions, yellow } = require('colorette');
+
+const packageExists = require('./utils/package-exists');
+const logger = require('./utils/logger');
+const { core, groups, coreFlagMap } = require('./utils/cli-flags');
 const argParser = require('./utils/arg-parser');
 const assignFlagDefaults = require('./utils/flag-defaults');
-const { writeFileSync } = require('fs');
-const { options: coloretteOptions } = require('colorette');
 const WebpackCLIPlugin = require('./plugins/WebpackCLIPlugin');
+const promptInstallation = require('./utils/prompt-installation');
 
 // CLI arg resolvers
 const handleConfigResolution = require('./groups/resolveConfig');
-const resolveMode = require('./groups/resolveMode');
-const resolveStats = require('./groups/resolveStats');
-const resolveOutput = require('./groups/resolveOutput');
-const basicResolver = require('./groups/basicResolver');
-const resolveAdvanced = require('./groups/resolveAdvanced');
 const toKebabCase = require('./utils/to-kebab-case');
 
+const WEBPACK_OPTION_FLAGS = core
+    .filter((coreFlag) => {
+        return coreFlag.group === groups.BASIC_GROUP;
+    })
+    .reduce((result, flagObject) => {
+        result.push(flagObject.name);
+        if (flagObject.alias) {
+            result.push(flagObject.alias);
+        }
+        return result;
+    }, []);
```

Comment:
Can we move it inside `resolveArgs`? no need decrease initial start

Reply:
done